superset-frontend/webpack.config.js

```diff
@@ -110,45 +108,29 @@ if (isDevMode) {
   plugins.push(new OptimizeCSSAssetsPlugin());
 }
 
-const BABEL_JAVASCRIPT_OPTIONS = {
-  presets: [
-    [
-      '@babel/preset-env',
-      {
-        useBuiltIns: 'usage',
-        corejs: 3,
-        loose: true,
-        shippedProposals: true,
-        modules: false,
-        targets: false,
-      },
-    ],
-    '@babel/preset-react',
-  ],
-  plugins: [
-    'lodash',
-    'react-hot-loader/babel',
-    '@babel/plugin-proposal-object-rest-spread',
-    '@babel/plugin-proposal-class-properties',
-    '@babel/plugin-syntax-dynamic-import',
-  ],
-};
-
-const BABEL_TYPESCRIPT_OPTIONS = {
-  presets: BABEL_JAVASCRIPT_OPTIONS.presets.concat([
-    '@babel/preset-typescript',
```

Comment:
`@babel/preset-typescript` is also removed from `package.json` as it is not used any more.